### PR TITLE
Cleaning up the components bits

### PR DESCRIPTION
**Pull request by hbons**

### CODE REVIEW

**Comment by netonjm** [comment]
to avoid conflicts this should be rebased and merged in bundle branch

File: FigmaSharp.NativeControls/FigmaSharp.NativeControls.Cocoa/Converters/ButtonConverter.cs [hunk 1]
```diff
@@ -88,6 +91,10 @@ public override IView ConvertTo(FigmaNode currentNode, ProcessedNode parent, Fig
                 if (group.name == "Disabled") {
 					button.Enabled = false;
                 }
+
+                if (group.name == "Default") {
+                    (button.NativeObject as NSButton).KeyEquivalent = "\r"; // Return key
+                }
```
Comment (netonjm) [inline FigmaSharp.NativeControls/FigmaSharp.NativeControls.Cocoa/Converters/ButtonConverter.cs]: nice!

File: FigmaSharp.NativeControls/FigmaSharp.NativeControls/Converters/FigmaControlsExtensions.cs [hunk 2]
```diff
@@ -46,23 +44,29 @@ public enum NativeControlType
...
+            // Buttons
             { "65e801586bf62f77a4d79085aaeb6e2bbd8a48ea", NativeControlType.ButtonStandard },
             { "52a371ab5df2ad1fe9f7ae1d51712ec9dc493325", NativeControlType.ButtonLarge },
             { "cc3e8e90f86a174cc22a27b7f8661f53f94b5d21", NativeControlType.ButtonSmall },
             { "101d73f2d75f295756728ee1fbb2ba3ce33fee5e", NativeControlType.ButtonLargeDark },
             { "4a606f23d957d508b11707faba119cbd7edf4565", NativeControlType.ButtonStandardDark },
             { "c252a811c880b9ab074d2ba63d3e279e7e3bad4a", NativeControlType.ButtonSmallDark },
 
-			{ "65722325467a63cc1dcb0851cd7cdc1bb3b8bd7c", NativeControlType.ButtonStandardDark }, //not sure why we have 2 instances
+            { "65722325467a63cc1dcb0851cd7cdc1bb3b8bd7c", NativeControlType.ButtonStandardDark }, //not sure why we have 2 instances
```
Comment (netonjm) [inline FigmaSharp.NativeControls/FigmaSharp.NativeControls/Converters/FigmaControlsExtensions.cs]: probably I added by mistake because an Id removed

**Comment by hbons** [comment]
This is old.

**Pull request closed without merging**